### PR TITLE
jq-1.5-foss-2016a

### PR DESCRIPTION
**Pull request by ysagon**

### CODE REVIEW

File: easybuild/easyconfigs/j/jq/jq-1.5-foss-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+name = 'jq'
+version = '1.5'
+homepage = 'https://stedolan.github.io/jq/'
+description = """jq is a lightweight and flexible command-line JSON processor."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://github.com/stedolan/jq/archive/']
+sources = ['%(name)s-%(version)s.tar.gz']
```
Comment (boegel) [inline easybuild/easyconfigs/j/jq/jq-1.5-foss-2016a.eb]: @ysagon Please include a `SHA256` checksum

File: easybuild/easyconfigs/j/jq/jq-1.5-foss-2016a.eb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+easyblock = 'ConfigureMake'
+
+name = 'jq'
+version = '1.5'
+homepage = 'https://stedolan.github.io/jq/'
```
Comment (boegel) [inline easybuild/easyconfigs/j/jq/jq-1.5-foss-2016a.eb]: add empty line above to match 'style' used in central easyconfigs

File: easybuild/easyconfigs/j/jq/jq-1.5-foss-2016a.eb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+source_urls = ['https://github.com/stedolan/jq/archive/']
+sources = ['%(name)s-%(version)s.tar.gz']
+
+dependencies = [('Bison', '3.0.4')]
+
+sanity_check_paths = {
+    'files': ["lib/libjq.so.1.0.4", 
+              "lib/libjq.a", 
+              "lib/libjq.la",
+              "bin/jq"],
```
Comment (boegel) [inline easybuild/easyconfigs/j/jq/jq-1.5-foss-2016a.eb]: no need to spread these across different lines imho; also, prefer only checking the `.so`, albeit via `SHLIB_EXT`:

```python
'files': ['bin/jq', 'lib/libjq.a', 'lib/libjq.%s' % SHLIB_EXT],

**Comment by migueldiascosta** [comment]
@ysagon, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Comment by boegel** [comment]
fwiw: superseded by #6632

**Pull request closed without merging**